qpmodel/Catalog.cs

```diff
@@ -51,7 +51,11 @@ public class ColumnDef
         readonly public ColumnType type_;
         public int ordinal_;
 
-        public ColumnDef(string name, ColumnType type, int ord) { name_ = name; type_ = type; ordinal_ = ord; }
+        public ColumnDef(string name, ColumnType type, int ord)
+        {
+            name_ = name.StartsWith('"') ? name : name.ToLower();
```

Comment:
can we add one utility function 
string normalizeName(string name) => name.StartsWith('"') ? name : name.ToLower();

to do the conversion?

Reply:
Sure. Let me do it right away. Will adding it in Utils.cs work?

Reply:
Looks like all source files use qpmodel.expr and expr.cs is used by all except PrestoStats.cs. So that should work.